src/ray/protobuf/common.proto

```diff
@@ -150,6 +150,14 @@ enum ErrorType {
   OBJECT_FETCH_TIMED_OUT = 14;
 }
 
+/// The information per ray error type.
+message RayErrorInfo {
+  oneof error {
+    RayException actor_init_failure = 1;
```

Comment:
In the future, we will include error different error info such as ActorDiedError metadata